Misc/NEWS.d/next/Library/2021-06-02-19-47-46.bpo-44289.xC5kuV.rst

```diff
@@ -0,0 +1 @@
+Keep argument file object's current position in tarfile.is_tarifle method.
```

Comment:
News entry can be more descriptive, so that users can easily tell if they need to update their codebase;
something along these lines:

Fix an issue with :meth:`~tarfile.is_tarfile` method when using *fileobj* argument: position
in the *fileobj* was advanced forward which made it unreadable with :meth:`tarfile.TarFile.open`.

Reply:
I tried to came up with something for new entry but your version is just right. I hope you don't mind me using it, do you?

Reply:
Of course, go ahead!